src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -1571,6 +1578,52 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
     return typedResult;
   };
 
+  private getTransferStatusCode = (result: string) => {
+    let status: TransferStatusCodes = TransferStatusCodes.TransferSuccess;
+    switch(result) {
```

Comment:
You can use the enum values themselves for cases instead of string constants (as long as you apply the change I suggested below in the enum definition). It will look a bit weird in the switch

```
case TransferStatusCode.TransferFailure: {
  status = TransferStatusCode.TransferFailure;
  break;
}
```

But it's better than hardcoding the strings here